app/actions/apps.ts

```diff
@@ -63,10 +63,7 @@ export function doAppCall<Res=unknown>(call: AppCall, intl: any): ActionFunc {
         } catch (err) {
             const msg = err.message || 'We found an unexpected error.';
             ephemeral(msg);
-            return {
-                type: AppCallResponseTypes.ERROR,
-                error: msg,
-            };
+            return {error: msg};
```

Comment:
Still confused whether doAppCall should return error or return data as a error call response.

Reply:
I've chosen the route of return data as a call response, so callers can find errors in `data.error` and `data.errors`